lib/matplotlib/backends/backend_svg.py

```diff
@@ -26,196 +26,120 @@
 from matplotlib.transforms import Affine2D, Affine2DBase
 from matplotlib import _png
 
-from xml.sax.saxutils import escape as escape_xml_text
+from xml.etree import ElementTree as etree
 
 backend_version = __version__
 
-# ----------------------------------------------------------------------
-# SimpleXMLWriter class
```

Comment:
That seems like a reasonable middle round.  I look forward to seeing it...  Thanks!